adonis-typings/application.ts

```diff
@@ -136,6 +137,8 @@ declare module '@ioc:Adonis/Core/Application' {
     raw: any
   }
 
+  export type AssetsDriver = 'vite' | 'encore'
```

Comment:
I think we should also allow for the `fake` asset driver here? Because that is what gets used during testing

Reply:
Oops. done